examples/learn/calls/custom_client/mistral/base_message_param.py

```diff
@@ -1,7 +1,20 @@
+import os
+
 from mirascope.core import BaseMessageParam, mistral
-from mistralai.client import MistralClient
+from mistralai import Mistral
 
 
-@mistral.call("mistral-large-latest", client=MistralClient())
+@mistral.call(
+    "mistral-large-latest",
+    client=Mistral(api_key=os.environ.get("MISTRAL_API_KEY", "")),
```

Comment:
ok, so after further thought I think that when using Mistral with Mirascope we should have a helper utility for getting the api key.

I noticed that their client accepts a function, so we could do something like `client=Mistral(mistral.load_api_key)`?

seems more convenient both internally + for users using a custom client, WDYT?

Reply:
hmm, actually just ignore this lol

we only load it internally once in setup call, otherwise I think it actually makes more sense for the user to just use `os` since that's what the mistral docs show and it's standard anyway